web/src/repo/blob/BlobPage.tsx

```diff
@@ -292,6 +282,16 @@ export class BlobPage extends React.PureComponent<Props, State> {
                         </div>
                     </div>
                 )}
+                {renderMode === 'code' && (
```

Comment:
```suggestion
                {/* Render the (unhighlighted) blob also in the case highlighting timed out */}
                {renderMode === 'code' && (
```
Prevents someone accidentally breaking this in the future